.github/scripts/build.sh

```diff
@@ -20,6 +20,10 @@ if [ "$RUN_BOOTSTRAP" = 1 ]; then
     .github/scripts/install-dependencies.sh
 fi
 
+if [ "$UPDATE_SOURCE" = 1 ]; then
```

Comment:
Why defining a new variable to enable the source update is needed? From my perspective using `RUN_BOOTSTRAP` for it is better.

Reply:
I would expect bootstrap should be needed less frequently and usually just once.
Update can happen every day to sync to the latest changes.